awx/sso/fields.py

```diff
@@ -1,58 +1,19 @@
 import collections
 import copy
-import inspect
 import json
 import re
 
 import six
 
-# Python LDAP
-import ldap
-import awx
-
 # Django
 from django.utils.translation import gettext_lazy as _
 
-# Django Auth LDAP
-import django_auth_ldap.config
-from django_auth_ldap.config import LDAPSearch, LDAPSearchUnion
-
 from rest_framework.exceptions import ValidationError
 from rest_framework.fields import empty, Field, SkipField
 
-# This must be imported so get_subclasses picks it up
-from awx.sso.ldap_group_types import PosixUIDGroupType  # noqa
```

Comment:
While working on the unit tests for the LDAP fields, a question I ran into is whether we still want to use this custom group type as I don't see it in DAB. The test for it is currently commented out as a result. Would we want to import this group back into DAB so it's picked up by get_subclasses there?